evaluators/rank/ReciprocalRankEvaluator/__init__.py

```diff
@@ -0,0 +1,31 @@
+from typing import Sequence, Union
+
+from jina.executors.evaluators.rank import BaseRankingEvaluator
+
+class ReciprocalRankEvaluator(BaseRankingEvaluator):
+    """
+    :class:`ReciprocalRankEvaluator` Gives score as per reciprocal rank metric.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
We need to see if we want to put the `eval_at` default here or in the `config.yml`. I would put it here and pass it to the `super().__init__()` call